### DISCUSSION TITLE
Discord webhook notifications

### DISCUSSION THREAD

**Discussion by Gummibeer** [spatie/laravel-backup#1282] [Ideas] [answered]
The usage of discord increases a lot during the last months. Two years ago there was already an "issue" #637 - I would like to ask again if you would merge a built-in discord webhook notification driver?
After first check it wouldn't come with any dependency (not even optional) as laravels http client should be enough.

I think that, primary for smaller and open-source teams, discord gets more usage than slack does. So there would be a user group benefiting from that.
Discord message building is also as simple as a string as they don't support attachments and so on - or I haven't found it.

**Comment by freekmurze** [comment, marked as answer]
Hi Tom,

great suggestion, I would accept a PR that adds a light channel / message for Discord. You could steal some of the code [in this blog post at Flare](https://flareapp.io/blog/9-flare-can-now-notify-you-via-discord-and-microsoft-teams)
